### ISSUE TITLE
Go to custom command definition / Custom command not found

### ISSUE THREAD

**Issue by thaianalps** [90poe/vscode-cy-helper#24]
**Bug or problem**
When I use "Cypress: Go to custom command definition" this message is displayed: "Custom command not found"

**My .vscode/setting.js**
```
{
    "cypressHelper.includeAnnotationForCommands": true,
    "cypressHelper.customCommandsFolder": "cypress/support",
    "cypressHelper.typeDefinitionFile": "cypress/support/customCommands.d.ts",
    "cypressHelper.fixtureAutocompletionCommands": [
        "fixture"
    ],
    "cypressHelper.menuItems": {
        "OpenSingleTest": true,
        "OpenSpecFile": true,
        "GenerateCustomCommandTypes": true,
        "GoToCustomCommand": true,
        "FindCustomCommandReferences": true,
        "FindStepDefinitionReferences": true
    },
}
```

**Expected behavior**
I expected it to be forwarded to the custom command.

**Environment ():**
- Extension version:                               4.766
- VS Code version:                                1.45.4
- OS:                                                     Linux (Mint):

Prints:

![screenshot_2](https://user-images.githubusercontent.com/13646613/83145730-a7aa9100-a0cb-11ea-9a5d-c3f709968aa8.png)
![screenshot](https://user-images.githubusercontent.com/13646613/83145499-3ec31900-a0cb-11ea-97c5-08a94e755c23.png)
![screenshot_1](https://user-images.githubusercontent.com/13646613/83145501-3ff44600-a0cb-11ea-9258-1f06af288faa.png)

**Comment by Shelex** [comment]
Hello @thaianalps !
Thank you for posting an issue.

It was not reproducing on OSX or Windows so I have checked VirtualBox with Linux Mint installation - and actually got it. That is because of sanitising absolute paths by removing trailing slashes. It seems that on linux distros the first slash is needed for accessing files, so I have provided a fix in version 0.6.2.

Could you please update and check if it works for you?

**Comment by thaianalps** [comment]
Worked perfectly!!!!

Thanks a lot!
